modules/tsb/mp/manifests/tsb/managementplane-values.yaml.tmpl

```diff
@@ -49,11 +38,24 @@ spec:
           signingKey: private.key
       signingKeysSecret: iam-signing-key
   components:
+    frontEnvoy:
+      port: 443
+    webUI:
```

Comment:
I think this was misplaced outside the `components` field.

Reply:
any reason behind removal of ```SESSION_AGE_IN_MINUTES``` or is it wrong?